homu/server.py

```diff
@@ -264,6 +264,8 @@ def callback():
 
 def rollup(user_gh, state, repo_label, repo_cfg, repo):
     user_repo = user_gh.repository(user_gh.user().login, repo.name)
+    if user_repo is None:
+        return 'You must have a fork of rust-lang/rust named rust under your user account.'  # noqa
```

Comment:
Unfortunately there's no way to test if this error works without merging. If someone has a dummy GitHub account for testing this sort of stuff, I would appreciate if you could test to make sure the error works!